flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/nfa/MigrationUtils.java

```diff
@@ -131,7 +131,7 @@ static <T> Queue<ComputationState> deserializeComputationStates(
 
             computationStates.add(
                     ComputationState.createState(
-                            state, nodeId, version, startTimestamp, startEventId));
+                            state, nodeId, version, startTimestamp, 0L, startEventId));
```

Comment:
Sometimes previousTimestamp is set to 0 and sometimes it set to -1. Could we keep it consistent?